Public/Src/Sandbox/Windows/DetoursServices/DetouredFunctions.cpp

```diff
@@ -1806,6 +1807,7 @@ BOOL WINAPI Detoured_CreateProcessW(
         injectedShim);
     if (injectedShim)
     {
+        Dbg(L"Injected shim for lpCommandLine='%s', returning 0x%08X from CreateProcessW", lpCommandLine, ret);

```

Comment:
>lpCommandLine [](start = 32, length = 13)

can lpCommandLine be NULL?

Reply:
Added additional check, having both lpApp and lpCmd null should not be possible but better to be defensive.